packages/apps/graph/generated-schema.graphql

```diff
@@ -239,6 +297,9 @@ type PageInfo {
   startCursor: String
 }
 
+"""The payload of a transaction."""
+union Payload = ContPayload | ExecPayload
```

Comment:
We might want to be a little more specific on this type name and call it `TransactionPayload` as in GraphQL it's hard to get around too generic names